modules/engine/src/model/model.ts

```diff
@@ -505,6 +505,7 @@ export class Model {
   /** Update uniform buffers from the model's shader inputs */
   updateShaderInputs(): void {
     this._uniformStore.setUniforms(this.shaderInputs.getUniformValues());
+    this.setBindings(this.shaderInputs.getBindings());
```

Comment:
Nit: not much reason for this comment other than symmetry:
```suggestion
    this.setBindings(this.shaderInputs.getBindingValues());
```